.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+<!--- PR title format: [GH#<Issue ID>][BZ<BZ ID>][<Jira ID>]: <Short description of your PR> --->
```

Comment:
If we're weighing in on # or -, I'd use a dash between BZ- or GH- or OSDOCS- etc. But as long as the separator isn't critical to this, then it's fine to keep this as is